core/simulations/helpers/executeSpell.ts

```diff
@@ -0,0 +1,71 @@
+import BigNumber from '../../src/bignumber';
+import hre from 'hardhat';
+import { TEST_NETWORK } from '../../helpers/constants';
+import getContract from '../../src/contracts';
+import getSigner from '../../src/signer';
+import { overwriteUintMapping } from '../../helpers/hardhat/slotOverwrite';
+import { MKR_NUMBER_OF_DIGITS } from '../../src/constants/UNITS';
+
+const ethers = hre.ethers;
+const provider = hre.network.provider;
+
+const getMaximumVotingPoverInMkr = async () => {
+    const chiefContract = await getContract(TEST_NETWORK, 'MCD_ADM');
+    const hatAddress = await chiefContract.hat();
+    const approvals = await chiefContract.approvals(hatAddress);
+    return new BigNumber(approvals._hex).shiftedBy(-MKR_NUMBER_OF_DIGITS);
+};
+
+const setGovernanceMkr = async (mkrAmount: BigNumber) => {
```

Comment:
from the name of the function it's not clear what it does (to me).
does it just set the mkr balance of the wallet? something else?

confusion comes from either unclear function name or from the fact that slot is just stated as direct argument instead of being put into the variable with descriptive name

Reply:
Well, it's hard to name it, since the thing it's going doesn't have a good real-world naming. Do you have a suggestion?

Reply:
After reading the contract I realised it was doing the same thing as `addMkrToBalance` so I removed it. Thanks for questioning it 🙂